File: 189-rotate-array/189-rotate-array.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        k = k if len(nums) > k else k%len(nums)
        temp = nums[-k:]
        temp = temp + nums[:-k]
        nums[:] = temp

File: 189-rotate-array/189-rotate-array.go
func rotate(nums []int, k int)  {
    k = k%len(nums)
    temp := append(nums[len(nums)-k:], nums[:len(nums)-k]...)
    for i:=0;i<len(nums);i++{
        nums[i] = temp[i]
    }
}